tests/test_todo.py

```diff
@@ -0,0 +1,84 @@
+from todo_app.data.session_items import *
+
+class TestGetCards_ToDo:
```

Comment:
I'm ignoring the tests in this PR as I suspect they're just on-route to being finished as part of exercise 3